airflow/providers/google/cloud/operators/gcs_to_gcs.py

```diff
@@ -40,32 +40,27 @@ class GCSToGCSOperator(BaseOperator):
     :param source_bucket: The source Google Cloud Storage bucket where the
          object is. (templated)
     :type source_bucket: str
-    :param source_object: The source name of the object to copy in the Google cloud
+    :param source_object: A list of prefix of the objects to copy in the Google cloud
         storage bucket. (templated)
-        You can use only one wildcard for objects (filenames) within your
-        bucket. The wildcard can appear inside the object name or at the
-        end of the object name. Appending a wildcard to the bucket name is
-        unsupported.
-    :type source_object: str
+    :type source_object: List[str]
```

Comment:
should be List [str] or str

But maybe it would be better to keep both source_object and source_objects parameters (source_object = str, source_objects=List[str]) and check that only one is set? 

I think that would be still backwards compatible, but more "accurate" with naming.

But I leave it up to you - I am ok with both approaches (as long as we have correct type).

Reply:
Nice. I'll take that approach of source_object and source_objects. Thanks a lot